CSV/sample-entries-boxes.csv

```diff
@@ -22,6 +22,7 @@ esds,Elementary stream descriptor,General,MP4v2
 hvcC,HEVC Configuration,Video,NALu Video
 hvtC,HEVC Tile Configuration,Video,NALu Video
 icam,Intrinsic camera parameters,Video,NALu Video
+jpgC,JPEG Configuration,Still Image, HEIF
```

Comment:
"Still Image" needs to resolve into handlers.csv, and the space after the comma deleted

Reply:
`pict` is indeed registered for "Image Item" and "Image sequences". It may be time to split the handler list into 2 categories, meta handlers and track handlers.

Reply:
try copying the Description field, i.e. "Image Item and Image sequences"